test/api/buffer/receivers/create/003-missing-storage-token-400/expected-response.json

```diff
@@ -0,0 +1,6 @@
+{
+  "statusCode": 400,
+  "error": "buffer.missingField",
+  "message": "\"X-StorageApi-Token\" is missing from header.",
+  "exceptionId": "keboola-buffer-%s"
```

Comment:
Mělo by to vracet spíš 401 Unauthorized, ale to asi generuje Goa a muselo by se to tam nějak nahackovat?

Reply:
Ono nestačí přidat nový error typ se status code `401`? Proč by se to muselo hackovat?

Reply:
Ono to berie ako chybajuci parameter, za mna to ma svoju logiku.
Nespravny token je `401`, nechal by som to tak.

Samozrejme da sa to poriesit, ak chcete.

Honza: Goa ma priamo podporuje pre rozne typy auth.

Reply:
Myslel jsem to tak, že to nekontrolujeme v kódu jestli ta hlavička je v requestu a nevyhazujeme tak tu chybu manuálně. Tak se to asi musí nějak potěšit v tom designu, ale možná je to snadný, jen já nevím na první pohled jak. 🙂

Reply:
Ked uz tak asi by to mala byt `403` (?)
https://stackoverflow.com/questions/4301877/http-status-code-for-missing-authentication

Reply:
No 401 se používá pravé když chybí autorizace

Reply:
> 401 Unauthorized
> Similar to 403 Forbidden, but specifically for use when authentication is required and has failed or has not yet been provided.

Reply:
^^^ tam pisu ale, ze sa to tyka iba `www-auth` `http://user:pass@...`, ale prakticky je mi to jedno

Reply:
A jo fakt. Hmm, dívám se že jiný API (včetně třeba AWS API Gateway) řeší chybějící token 403kou. Ale tady se zase píše https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403 : "re-authenticating makes no difference", tak nevím. Vždycky mi přišlo že je v těch kódech bordel a snažil jsem se jich minimalisticky používat co nejmíň, tak to píšu jen proto že jsi tady k tomu vzal opačný přístup. 😄 Tak bych to neřešil (obzvlášť ne v tomhle PR.)

Reply:
ok, mozes zalozit task do cooldown, a zoradime si to, podla toho co ma pre nas najvacsi prinos